### PR TITLE
[WIP] Sergal Redesign and subspecies

### PR DESCRIPTION
**Pull request by Poojawa**
Henlo. 

Making new sergs for people who want to look like sergs and not melted face people.

current general progress:

TODO:
- [ ] Sprite markings and hair styles to match up with new head shape.
- [ ] Convert uniforms and suits to new body type
- [ ] Convert shoes
- [ ] align head gear
- [ ] align hand held items
- [ ] polish

### CODE REVIEW

**Comment by Arokha** [comment]
Wedgewedgewedge

**Comment by KasparoVy** [comment]
Woah woah, what melted face? Was there something FUBAR with the sprites?
The tail's lost a lot of its floof, and the ears! I agree with digitigrade legs, but I really don't know what the problem was!

Why?
virgo ss13 what melted face 20200607

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 1]
```diff
@@ -61,6 +61,16 @@
 		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right)
 		)
 
+/datum/species/sergal/nsergal
+	//lankier appearing
+	name = SPECIES_NSERGAL
+	name_plural = "Low Gee Sergals"
+	icobase = 'icons/mob/human_races/r_nsergal.dmi'
+	deform = 'icons/mob/human_races/r_def_nsergal.dmi'
+	blurb = "After decades of adaptation in all lines of work, the usual heavier set sergals have developed into a more lanky build. \
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm:70]: usually*

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 2]
```diff
@@ -61,6 +61,16 @@
...
+			 Extended time in micro gravity environments have promoted more flexibility and little loss of wiry strengths. \
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm:71]: some loss of wiry strength

**Comment by KasparoVy** [comment]
Hell yeah

**Comment by Poojawa** [comment]
not deleting the branch, but I just haven't had any motivation to continue this, sorry!

**Comment by KasparoVy** [comment]
Aw. Thanks for your efforts anyway, hope you been well otherwise.

**Pull request closed without merging**